lld/COFF/InputFiles.cpp

```diff
@@ -1061,6 +1061,11 @@ void BitcodeFile::parse() {
     } else {
       sym = ctx.symtab.addRegular(this, symName, nullptr, fakeSC, 0,
                                   objSym.isWeak());
+      // If one LTO object defines a __imp_<foo> symbol, and another LTO
```

Comment:
The use case you've described is very specific, can you please lead with a higher-level comment about what this code does? As a suggestion, you can start with:
// Model all symbols with the `__imp_` prefix as having external references. If one LTO object ...

Reply:
Sure, done - thanks!